source/OutfitterPanel.cpp

```diff
@@ -359,19 +358,15 @@ void OutfitterPanel::Buy(bool alreadyOwned)
 	int modifier = Modifier();
 	for(int i = 0; i < modifier && CanBuy(alreadyOwned); ++i)
 	{
-		// Special case: maps.
+		// Maps are not transferrable; when bought, they make nearby planets known to the player.
```

Comment:
We can keep the existing comment:
```suggestion
		// Special case: maps.
```

Reply:
Done